deform/field.py

```diff
@@ -278,12 +278,15 @@ def clone(self):
         attribute of the node is not cloned; instead the field
         receives a new order attribute; it will be a number larger
         than the last rendered field of this set.  The parent of the cloned
-        node will become ``None`` unconditionally."""
+        node will be retained by the clone."""
         cloned = self.__class__(self.schema)
         cloned.__dict__.update(self.__dict__)
         cloned.order = next(cloned.counter)
         cloned.oid = 'deformField%s' % cloned.order
-        cloned._parent = None
```

Comment:
Can you add a comment here why this is done as it is being done?

Reply:
I have not looked at this in years, so have forgotten anything I knew about this.
I am just now starting to look at it again.  I think [this comment][] explains the need for being able to access the parent of the cloned field.

[this comment]: <https://github.com/Pylons/deform/pull/187#issuecomment-25821788> (thread on #187)